mods/usermanager/test/usermanager.unit.test.ts

```diff
@@ -12,14 +14,47 @@ const sandbox = sinon.createSandbox()
 describe('@fonos/usermanager', () => {
   afterEach(() => sandbox.restore())
 
-  it.skip('needs testing', async () => {
+  it('checks the requests parameters', async () => {
+
+    const initStub = sandbox.stub(FonosService.prototype, 'init').returns()
+    const serviceStub = sandbox
+      .stub(FonosService.prototype, 'getService')
+      .returns({
+        createUser: () => {
+          return {
+            sendMessage: () =>
+              Promise.resolve({
+                getFirstName: () => 'Test',
+                getLastName: () => 'Test',
+                getEmail: () => 'test@gmail.com',
+                getAccessKeyId: () => '70lkv20G5MCgUIKYJI6Z',
+                getRole: () => 'USER',
+                getCreateTime: () => '2020-10-10',
+                getUpdateTime: () => '2020-10-10',
+                getStatus:() => 'ACTIVE'
+              })
+          }
+        }
+      })
+
+    const callStub = sandbox.spy(UserManager.prototype, 'createUser')
     const userManager = new UserManager()
-    await userManager.createUser(
-      {
-        firstName: "test",
-        lastName: "test",
-        email : "testing@gmail.com"
-      }
-    )
+    const result = await userManager.createUser({
+      firstName: "Test",
+      lastName: "Test",
+      email : "testing@gmail.com"
+    })
+
+
+    expect(initStub).to.be.calledOnce
+    // Once in the constructor and one in the call function
+    expect(serviceStub).to.be.calledTwice
+    expect(callStub).to.be.calledOnce
+    expect(result).to.have.property('firstName')
```

Comment:
I recommend check the value of the property as well. For example

```
expect(result).to.have.property('firstName').to.be.equal("Test")
```

This will help us avoid error while computing the values.